### PR TITLE
Add INS output for Vectornav [CSEMB-1898]

### PR DESCRIPTION
**Pull request by KevinQLuo**
Adding `/vectornav/INS` topic and a new ROS message to provide INS solution for Vectornav

Though we may not use Vectornav for SCOUT, I hope this INS support will be useful in some ways.

### CODE REVIEW

**Comment by KevinQLuo** [comment]
Sample INS Output published on `/vectornav/INS` topic:

```
header: 
  seq: 209971
  stamp: 
    secs: 1613767099
    nsecs: 599877676
  frame_id: "vectornav"
time: 506317.572877
week: 2145
utcTime: 161306215921680917
insStatus: 133
yaw: -99.9113769531
pitch: -0.27863484621
roll: 0.549136340618
latitude: 37.7732199285
longitude: -121.991589815
altitude: 159.939641539
nedVelX: -0.0350817777216
nedVelY: -0.0144447609782
nedVelZ: -0.122825734317
attUncertainty: [19.935606002807617, 0.09668409079313278, 0.09626581519842148]
posUncertainty: 2.01280450821
velUncertainty: 0.153650537133
```

File: src/main.cpp [hunk 1]
```diff
@@ -229,19 +232,22 @@ int main(int argc, char *argv[])
...
+            | TIMEGROUP_GPSWEEK
+            | TIMEGROUP_TIMEUTC,
             IMUGROUP_NONE,
             GPSGROUP_NONE,
             ATTITUDEGROUP_YPRU, //<-- returning yaw pitch roll uncertainties
             INSGROUP_INSSTATUS
             | INSGROUP_POSLLA
-            | INSGROUP_POSECEF
-            | INSGROUP_VELBODY
-            | INSGROUP_ACCELECEF,
```
Comment (neelfirst) [inline src/main.cpp:244]: Do you want to push back to the public repository? If so we should find a way to have ECEF and NED coexist.

File: src/main.cpp [hunk 2]
```diff
@@ -229,19 +232,22 @@ int main(int argc, char *argv[])
             ASYNCMODE_PORT1,
             SensorImuRate / async_output_rate,  // update rate [ms]
             COMMONGROUP_QUATERNION
+			| COMMONGROUP_YAWPITCHROLL
```
Comment (neelfirst) [inline src/main.cpp]: nit: spacing

**Review by neelfirst — COMMENTED** [review]
I'm wondering if you want to target the public repo, in which case we need to maximize compatibiltiy with the existing code. I think it's a worthy goal.

**Review by twslankard — APPROVED** [review]
LGTM.

**Review by neelfirst — APPROVED** [review]

**Pull request merged**